NMS-13330: Add out of band monitoring content to main user documentation (#3453)
OpenNMS/opennms: docs/modules/operation/pages/bsm/business-service-hierarchy.adoc

= Business service hierarchy

Business services can depend on each other and together build a business service hierarchy.

> Visualize the hierarchy using the Topology User Interface with the business services view.

Author: I just did a search and replace on "BS" to change it to business services, but didn't reference the UI. Thanks for point this out. Since this is a menu option, then the style should be Bold. I'll update.

The operational status of a business service is ultimately calculated from alarms and their serverity.

> Ues a To define the class of alarms a reduction key is used and is represented as an edge of a business service.

Reviewer: Honestly, this sentence seems out of place in the paragraph and might be better off removed from here.
```suggestion
Use a reduction key to define a class of alarms as an edge to a business service.
```

.Business Service Hierarchy components
image::bsm/02_bsm-service-hierarchy.png[]

(1) A top-level business service that depends on other business services, monitored services, and alarms (referenced by reduction key).

> (2) Business service as child and the operational status are used as input for the top-level business service.

Reviewer: ```suggestion
(2) Business service used as child where its operational status is used as input for the top-level business service.
```

(3) IP service edge used as an input with auto-generated reduction keys for node down, interface down, and node lost service.

> (4) Reduction key edge used as an input to the top-level business service, which references just a node-lost service of a page sequence monitor for the user login.

Reviewer: ```suggestion
(4) Reduction key edge used as an input to the top-level business service, which only references a node lost service of a page sequence monitor for the user login.
```

> To add or remove an additional selected BS or edge to focus use, in the context menu Add To Focus or Remove From Focus.

Reviewer: ```suggestion
To add or remove an additional selected business service or edge to focus use, in the context menu, select Add To Focus or Remove From Focus.
```

> If you want to have a specific BS or edge as a single focus use Set as Focal Point.

Reviewer: ```suggestion
If you want to have a specific business service or edge as a single focus, select Set as Focal Point.
```
